src/main/java/net/ccbluex/liquidbounce/ui/client/fontmanager/GuiFontManager.kt

```diff
@@ -0,0 +1,221 @@
+    private lateinit var removeButton: GuiButton
+    private lateinit var textField: GuiTextField
+
+    override fun initGui() {
+        val textFieldWidth = (width / 8).coerceAtLeast(70)
+        textField = GuiTextField(2, mc.fontRendererObj, width - textFieldWidth - 10, 10, textFieldWidth, 20)
+        textField.maxStringLength = Int.MAX_VALUE
+
+        val startPositionY = 22
+        addButton = +GuiButton(ADD_BTN_ID, width - 80, startPositionY + 24 * 1, 70, 20, translationButton("add"))
+        removeButton = +GuiButton(REMOVE_BTN_ID, width - 80, startPositionY + 24 * 2, 70, 20, translationButton("remove"))
+        +GuiButton(EDIT_BTN_ID, width - 80, startPositionY + 24 * 3, 70, 20, translationButton("fontManager.edit"))
+
+        fontListView = GuiList(this).apply {
+            registerScrollButtons(7, 8)
+        }
+    }
+
+    override fun handleMouseInput() {
+        super.handleMouseInput()
+        fontListView.handleMouseInput()
+    }
+
+    override fun mouseClicked(mouseX: Int, mouseY: Int, mouseButton: Int) {
+        textField.mouseClicked(mouseX, mouseY, mouseButton)
+        super.mouseClicked(mouseX, mouseY, mouseButton)
+    }
+
+    public override fun keyTyped(typedChar: Char, keyCode: Int) {
+        if (textField.isFocused) {
+            textField.textboxKeyTyped(typedChar, keyCode)
+        }
+
+        when (keyCode) {
+            // Go back
+            Keyboard.KEY_ESCAPE -> mc.displayGuiScreen(prevGui)
+
+            // Go one up in account list
+            Keyboard.KEY_UP -> fontListView.selectedSlot -= 1
+
+            // Go one down in account list
+            Keyboard.KEY_DOWN -> fontListView.selectedSlot += 1
+
+            // Go up or down in account list
+            Keyboard.KEY_TAB -> fontListView.selectedSlot += if (Keyboard.isKeyDown(Keyboard.KEY_LSHIFT)) -1 else 1
+
+            // Login into account
+            Keyboard.KEY_RETURN -> fontListView.elementClicked(fontListView.selectedSlot, true, 0, 0)
+
+            // Scroll account list
+            Keyboard.KEY_NEXT -> fontListView.scrollBy(height - 100)
+
+            // Scroll account list
+            Keyboard.KEY_PRIOR -> fontListView.scrollBy(-height + 100)
+
+            // Add account
+            Keyboard.KEY_ADD -> actionPerformed(addButton)
+
+            // Remove account
+            Keyboard.KEY_DELETE, Keyboard.KEY_MINUS -> actionPerformed(removeButton)
+
+            else -> super.keyTyped(typedChar, keyCode)
+        }
+    }
+
+    override fun drawScreen(mouseX: Int, mouseY: Int, partialTicks: Float) {
+        assumeNonVolatile {
+            drawBackground(0)
+            fontListView.drawScreen(mouseX, mouseY, partialTicks)
+            Fonts.fontSemibold40.drawCenteredString(translationMenu("fontManager"), width / 2f, 6f, 0xffffff)
+            val count = Fonts.customFonts.size
+            Fonts.fontSemibold35.drawCenteredString(
+                "$count Custom Font${if (count > 1) "s" else ""}",
```

Comment:
```suggestion
                "$count Custom Font${if (count != 1) "s" else ""}",
```
anything below 1 & above 1 is plural
also, this probably should be a i18n string lol